lib/enumerable/inject.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_inject(initial = nil, operation = nil)
+    return multiply_els(self, initial) if operation == :*
+
+    verified = verify_input(initial)
+
+    input = self.class == Range ? to_a : self
+    result = initial.nil? || initial.is_a?(Symbol) ? 0 : initial
+    if block_given?
+      start = verified[1]
```

Comment:
Could you investigate this? :thinking: 

Since `verify_input(initial)` will also return either `1` or `0` and not an `array`, why did we have `verified[1]` which will always result to `0`?

Reply:
Now it have 2 arguments, and the logic is different